### DISCUSSION TITLE
Dualmode... dualmode?

### DISCUSSION THREAD

**Discussion by mew905** [Lolliedieb/lolMiner-releases#320] [Q&A]
Can I combine dualmode etc with dualmode zil?

HiveOS won't let me run two of the same miners, even with different settings. I'm quite enjoying your application's capabilities but unsure if I can even do this.

Like can I use zilwallet.zilworker@zilstratum or whatever the format for the etc and eth dual stratum modes were?

Also concerning running zil + eth on a different pool.... Can it be done with DaggerHashimoto? I realize they both mine Eth, but Phoenixminer requires specifying Dagger or Ethash so I'm unsure here.

**Comment by ChesDonno** [comment]
In your lolminer.cfg
pick your primary coin 
algo=ETCHASH
pool=poolAddress:port
user=walletAddress.rigName

save cfg file

and when you're ready to run it 

./lolMiner --dualmode zil --dualdeveices (pick number of gpu you want to mine zil with) --dualstratum walletAddress.rigName@poolAddress:port
